### ISSUE TITLE
Tried creating a world and this happened :(

### ISSUE THREAD

**Issue by Peter226** [MovingBlocks/Terasology#5101]
version 5.3.0 build 11

https://pastebin.com/Z98Nbawc
[Terasology Error Report.log](https://github.com/MovingBlocks/Terasology/files/11320765/Terasology.Error.Report.log)

**Comment by Peter226** [comment]
(Failed to create it lol)

**Comment by skaldarnar** [comment]
Running engine:5.3.0 with Josharias Survival.

Some errors are logged, but since loading continued I don't think they are responsible for the crash here. Nevertheless, I'd be curious to investigate why this happens.

```
11:03:15.530 [main] ERROR o.t.g.m.s.JavaModuleClassLoader - Denied access to class (not allowed with this module's permissions): org.terasology.gestalt.module.sandbox.API
11:03:15.530 [main] ERROR o.t.g.m.s.JavaModuleClassLoader - Denied access to class (not allowed with this module's permissions): org.terasology.gestalt.module.sandbox.API
11:03:15.530 [main] ERROR o.t.g.m.s.JavaModuleClassLoader - Denied access to class (not allowed with this module's permissions): org.terasology.gestalt.module.sandbox.API
11:03:15.531 [main] ERROR o.t.g.m.s.JavaModuleClassLoader - Denied access to class (not allowed with this module's permissions): org.terasology.gestalt.module.sandbox.API
11:03:15.552 [main] ERROR o.t.e.u.gson.UriTypeAdapterFactory - URI type interface org.terasology.engine.core.Uri lacks String constructor
```

There are some more errors on `ComponentSystem` (I think these are artifacts of a half-finished migration):

```
11:03:18.407 [main] ERROR o.t.e.core.ComponentSystemManager - Cannot load BulletPhysics, must be a subclass of ComponentSystem
11:03:18.411 [main] ERROR o.t.e.core.ComponentSystemManager - Cannot load ModuleRendering, must be a subclass of ComponentSystem
```

and finally some warnings about the rendering graph, but the logs seems to be truncated:

```
11:03:20.188 [main] WARN  o.t.engine.rendering.dag.RenderGraph - WARNING: destination connection (Output:CoreRendering:finalHazeNode:BufferPair1(connected to CoreRendering:alphaRejectBlocksNode:BufferPair1, CoreRendering:opaqueBlocksNode:BufferPair1, CoreRendering:opaqueObjectsNode:BufferPair1, ;)) is already connected to ({CoreRendering:alphaRejectBlocksNode:BufferPair1=Input:CoreRendering:alphaRejectBlocksNode:BufferPair1(connected to CoreRendering:finalHazeNode:BufferPair1, ;), CoreRendering:opaqueBlocksNode:BufferPair1=Input:CoreRendering:opaqueBlocksNode:BufferPair1(connected to CoreRendering:finalHazeNode:BufferPair1, ;), CoreRendering:opaqueObjectsNode:BufferPair1=Input:CoreRendering:opaqueObjectsNode:BufferPair1(connected to CoreRendering:finalHazeNode:BufferPair1, ;)}
11:03:20.188 [main] WARN  o.t.engine.rendering.dag.RenderGraph - Trying to disconnect two nodes that aren'
```

@Peter226 are you sure these logs are complete? If you try it again, can you post the full log, most importantly the last parts at the very bottom?

**Comment by josephine-rueckert** [comment]
@skaldarnar The logs are complete, it's just a pastebin bug that they are shortened in the normal view. If you look at the raw view everything is there including the actual issue:
```
11:03:22.103 [main] ERROR o.t.engine.core.modes.StateLoading - Error while loading org.terasology.engine.core.modes.loadProcesses.SetupLocalPlayer@3b308104
java.lang.IllegalArgumentException: Out of bounds: (-33, -33) for region BlockArea[(-30, -30)...(30, 30)]
	at org.terasology.engine.world.generation.facets.base.BaseFacet2D.getWorldIndex(BaseFacet2D.java:41)
	at org.terasology.engine.world.generation.facets.base.BaseFieldFacet2D.getWorld(BaseFieldFacet2D.java:31)
	at org.terasology.core.world.generator.facetProviders.DensityNoiseProvider.process(DensityNoiseProvider.java:59)
	at org.terasology.engine.world.generation.ScalableFacetProvider.process(ScalableFacetProvider.java:10)
	at org.terasology.engine.world.generation.RegionImpl.getFacet(RegionImpl.java:41)
	at org.terasology.engine.logic.spawner.AbstractSpawner.findSpawnPosition(AbstractSpawner.java:46)
	at org.terasology.engine.logic.spawner.FixedSpawner.getSpawnPosition(FixedSpawner.java:36)
	at org.terasology.core.world.generator.worldGenerators.SimplexFacetedWorldGenerator.getSpawnPosition(SimplexFacetedWorldGenerator.java:51)
	at org.terasology.engine.logic.players.PlayerSystem.onConnect(PlayerSystem.java:143)
	at org.terasology.engine.logic.players.PlayerSystemMethodAccess.invoke(Unknown Source)
	at org.terasology.engine.entitySystem.event.internal.EventSystemImpl$ByteCodeEventHandlerInfo.invoke(EventSystemImpl.java:399)
	at org.terasology.engine.entitySystem.event.internal.EventSystemImpl.sendStandardEvent(EventSystemImpl.java:264)
	at org.terasology.engine.entitySystem.event.internal.EventSystemImpl.send(EventSystemImpl.java:255)
	at org.terasology.engine.core.bootstrap.eventSystem.AbstractEventSystemDecorator.send(AbstractEventSystemDecorator.java:67)
	at org.terasology.engine.network.NetworkEventSystemDecorator.send(NetworkEventSystemDecorator.java:54)
	at org.terasology.engine.core.bootstrap.eventSystem.AbstractEventSystemDecorator.send(AbstractEventSystemDecorator.java:67)
	at org.terasology.engine.recording.RecordingEventSystemDecorator.send(RecordingEventSystemDecorator.java:34)
	at org.terasology.engine.entitySystem.entity.internal.BaseEntityRef.send(BaseEntityRef.java:190)
	at org.terasology.engine.network.internal.NetworkSystemImpl.connectClient(NetworkSystemImpl.java:892)
	at org.terasology.engine.network.internal.NetworkSystemImpl.joinLocal(NetworkSystemImpl.java:325)
	at org.terasology.engine.core.modes.loadProcesses.SetupLocalPlayer.step(SetupLocalPlayer.java:29)
	at org.terasology.engine.core.modes.StateLoading.update(StateLoading.java:259)
	at org.terasology.engine.core.TerasologyEngine.tick(TerasologyEngine.java:512)
	at org.terasology.engine.core.TerasologyEngine.mainLoop(TerasologyEngine.java:472)
	at org.terasology.engine.core.TerasologyEngine.runMain(TerasologyEngine.java:448)
	at org.terasology.engine.core.TerasologyEngine.run(TerasologyEngine.java:414)
	at org.terasology.engine.Terasology.call(Terasology.java:190)
	at org.terasology.engine.Terasology.call(Terasology.java:70)
	at picocli.CommandLine.executeUserObject(CommandLine.java:1933)
	at picocli.CommandLine.access$1200(CommandLine.java:145)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2332)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2326)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2291)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2159)
	at picocli.CommandLine.execute(CommandLine.java:2058)
	at org.terasology.engine.Terasology.main(Terasology.java:139)
```

@Peter226 This is a sporadic we're currently facing and haven't been able to fix yet (see #5056 )

I'll close this as its a duplicate of that issue tracked in #5056

**Comment by jdrueckert** [comment]
@skaldarnar The logs are complete, it's just a pastebin bug that they are shortened in the normal view. If you look at the raw view everything is there including the actual issue:
```
11:03:22.103 [main] ERROR o.t.engine.core.modes.StateLoading - Error while loading org.terasology.engine.core.modes.loadProcesses.SetupLocalPlayer@3b308104
java.lang.IllegalArgumentException: Out of bounds: (-33, -33) for region BlockArea[(-30, -30)...(30, 30)]
	at org.terasology.engine.world.generation.facets.base.BaseFacet2D.getWorldIndex(BaseFacet2D.java:41)
	at org.terasology.engine.world.generation.facets.base.BaseFieldFacet2D.getWorld(BaseFieldFacet2D.java:31)
	at org.terasology.core.world.generator.facetProviders.DensityNoiseProvider.process(DensityNoiseProvider.java:59)
	at org.terasology.engine.world.generation.ScalableFacetProvider.process(ScalableFacetProvider.java:10)
	at org.terasology.engine.world.generation.RegionImpl.getFacet(RegionImpl.java:41)
	at org.terasology.engine.logic.spawner.AbstractSpawner.findSpawnPosition(AbstractSpawner.java:46)
	at org.terasology.engine.logic.spawner.FixedSpawner.getSpawnPosition(FixedSpawner.java:36)
	at org.terasology.core.world.generator.worldGenerators.SimplexFacetedWorldGenerator.getSpawnPosition(SimplexFacetedWorldGenerator.java:51)
	at org.terasology.engine.logic.players.PlayerSystem.onConnect(PlayerSystem.java:143)
	at org.terasology.engine.logic.players.PlayerSystemMethodAccess.invoke(Unknown Source)
	at org.terasology.engine.entitySystem.event.internal.EventSystemImpl$ByteCodeEventHandlerInfo.invoke(EventSystemImpl.java:399)
	at org.terasology.engine.entitySystem.event.internal.EventSystemImpl.sendStandardEvent(EventSystemImpl.java:264)
	at org.terasology.engine.entitySystem.event.internal.EventSystemImpl.send(EventSystemImpl.java:255)
	at org.terasology.engine.core.bootstrap.eventSystem.AbstractEventSystemDecorator.send(AbstractEventSystemDecorator.java:67)
	at org.terasology.engine.network.NetworkEventSystemDecorator.send(NetworkEventSystemDecorator.java:54)
	at org.terasology.engine.core.bootstrap.eventSystem.AbstractEventSystemDecorator.send(AbstractEventSystemDecorator.java:67)
	at org.terasology.engine.recording.RecordingEventSystemDecorator.send(RecordingEventSystemDecorator.java:34)
	at org.terasology.engine.entitySystem.entity.internal.BaseEntityRef.send(BaseEntityRef.java:190)
	at org.terasology.engine.network.internal.NetworkSystemImpl.connectClient(NetworkSystemImpl.java:892)
	at org.terasology.engine.network.internal.NetworkSystemImpl.joinLocal(NetworkSystemImpl.java:325)
	at org.terasology.engine.core.modes.loadProcesses.SetupLocalPlayer.step(SetupLocalPlayer.java:29)
	at org.terasology.engine.core.modes.StateLoading.update(StateLoading.java:259)
	at org.terasology.engine.core.TerasologyEngine.tick(TerasologyEngine.java:512)
	at org.terasology.engine.core.TerasologyEngine.mainLoop(TerasologyEngine.java:472)
	at org.terasology.engine.core.TerasologyEngine.runMain(TerasologyEngine.java:448)
	at org.terasology.engine.core.TerasologyEngine.run(TerasologyEngine.java:414)
	at org.terasology.engine.Terasology.call(Terasology.java:190)
	at org.terasology.engine.Terasology.call(Terasology.java:70)
	at picocli.CommandLine.executeUserObject(CommandLine.java:1933)
	at picocli.CommandLine.access$1200(CommandLine.java:145)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2332)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2326)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2291)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2159)
	at picocli.CommandLine.execute(CommandLine.java:2058)
	at org.terasology.engine.Terasology.main(Terasology.java:139)
```

@Peter226 This is a sporadic we're currently facing and haven't been able to fix yet (see #5056 )

I'll close this as its a duplicate of that issue tracked in #5056
